src/test/app/claims/models/claim.ts

```diff
@@ -63,6 +66,20 @@ describe('Claim', () => {
         expect(claim.eligibleForCCJ).to.be.equal(false)
       })
     })
+
+    context('totalAmountTillToday', () => {
+      it('should return the correct amount with interest until today', () => {
+        expect(sampleClaimObj.totalAmountTillToday).to.equal(225)
```

Comment:
Not configuring an instance before testing makes this test fragile. It is also difficult to see what made the result be `255`.

Reply:
Do you think the test will pass tomorrow?

Reply:
This test couldn't be more fake. It says `amount with interest until today` while sample object has no interest selected. How it is supposed to calculate interest if claimant did not wish to have interest added?

Reply:
Yes I should have checked the sample claim test data